mtp_api/apps/security/tests/test_views.py

```diff
@@ -1267,6 +1273,138 @@ def test_check_filtering_by_actioned_by(self):
             self.assertIsNotNone(item['actioned_by'])
 
 
+class PatchCheckTestCase(BaseCheckTestCase):
+    """
+    Tests related to patching security checks.
+    """
+
+    def test_patch(self):
+        """
+        Tests related to patching a single security check.
+        """
+        check = Check.objects.first()
+        assigned_to_user = self.security_fiu_users[0]
+
+        auth = self.get_http_authorization_for_user(self._get_authorised_user())
+        response = self.client.patch(
+            reverse(
+                'security-check-detail',
+                kwargs={'pk': check.pk},
+            ),
+            data={
+                'assigned_to': assigned_to_user.id
+            },
+            format='json',
+            HTTP_AUTHORIZATION=auth,
+        )
+        self.assertEqual(response.status_code, http_status.HTTP_200_OK)
+
+        actual_check_data = response.json()
+        actual_check_data['assigned_to'] = assigned_to_user.id
```

Comment:
is this (and line below) not already set in the response? it seems a bit odd to issue a patch instruction and then modify the response as well to match… doesn't that mean this test will always pass because the response is manually tweaked?

Reply:
Well this is embarrassing, that's supposed to be an assertion